### DISCUSSION TITLE
Tyring to create program in program service for cokreat

### DISCUSSION THREAD

**Discussion by shubhambansaltarento** [Sunbird-coKreat/Community#63] [Q&A]
I am using curl  request (postman collection attached)  to create a program in cokreat but it not giving me any response, please help 
[program service.postman_collection.zip](https://github.com/Sunbird-coKreat/Community/files/11100283/program.service.postman_collection.zip)

**Comment by mmanimegalai** [comment]
Tagging @arun1205 @vaivk369

**Reply by arun1205** [reply]
@mmanimegalai  Already spoke to @shubhambansaltarento , @vaivk369 already had a first round call it seems

**Reply by vaivk369** [reply]
Can you please confirm if you have update the environment file for correct configuration of Postgres database and also the authorisation token

**Reply by shubhambansaltarento** [reply]
Yes, I have updated the environment file for the local postgres database,  but still it is not up and running
